src/themes/default.js

```diff
@@ -168,13 +168,48 @@ export const fontWeight = {
   bold: '700'
 };
 
-export const breakpoints = {
-  untilMedium: '(max-width: 450px)',
-  medium: 451,
-  mediumToBig: '(min-width: 451px) and (max-width: 800px)',
-  big: 801,
-  bigToHuge: '(min-width: 801px) and (max-width: 1401px)',
-  huge: 1401
+export const grid = {
+  default: {
+    breakpoint: 'default',
+    cols: 12,
```

Comment:
This is also something I wanted to discuss.

I think it's easier if we always go with 12 columns with the same gutter in order to create consistency.

It would look like something like this:

```jsx
// 25% width on larger screens, 50% on mobile
<Col span={{ default: 3, untilKilo: 6 }} />
```

We will still follow the mentality of having different column limits per breakpoint, but you remove the need of thinking about each one individually. (We still cover the design needs in the end)

What do you think @felixjung?

Reply:
You mean having twelve columns at every breakpoint in the implementation but adjusting the `span` so it simulates having fewer? It sure makes the grid much simpler, implementation wise, but it also adds some mental overhead when converting the designs to the grid. At the same time, if we introduce standard components for our default layouts, I think this is a non-issue. So from my side, I'd say go for it.